integration-cli/docker_cli_daemon_test.go

```diff
@@ -481,6 +481,31 @@ func TestDaemonUpgradeWithVolumes(t *testing.T) {
 	logDone("daemon - volumes from old(pre 1.3) daemon work")
 }
 
+// GH#11320 - verify that the daemon exits on failure properly
+func TestDaemonExitOnFailure(t *testing.T) {
```

Comment:
@cpuguy83 I assumed he meant that it would be clearer to someone down the road in case `--bridge` and `--bip` ever end up being non-conflicting this test would start failing and some other "error case" would need to be used?